beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -226,8 +226,13 @@ func createLightClientBootstrapDeneb(ctx context.Context, state state.BeaconStat
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get current sync committee proof")
 	}
-
-	branch := make([]string, fieldparams.NextSyncCommitteeBranchDepth)
+	var branch []string
+	switch block.Version() {
```

Comment:
for Electra the value of `range currentSyncCommitteeProof` is coming out to be 6 whereas in Deneb it is 5. I am not sure if this is a good way to define the size. Open to suggestions

Reply:
We should have a new constant for that - `NextSyncCommitteeBranchDepthElectra`